benchmark/Utils.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace MongoDB\Benchmark;
+
+use MongoDB\Client;
+use MongoDB\Collection;
+
+use function getenv;
+
+final class Utils
+{
+    private static ?Client $client;
+    private static ?Collection $collection;
+
+    public static function getClient(): Client
+    {
+        return self::$client ??= self::createClient();
+    }
+
+    public static function getCollection(): Collection
+    {
+        return self::$collection ??= self::createCollection();
+    }
+
+    public static function getUri(): string
+    {
+        return getenv('MONGODB_URI') ?: 'mongodb://localhost:27017/';
+    }
+
+    public static function getDatabase(): string
+    {
+        return getenv('MONGODB_DATABASE') ?: 'phplib_test';
+    }
+
+    private static function createClient(): Client
+    {
+        return new Client(self::getUri());
+    }
+
+    private static function createCollection(): Collection
+    {
+        return self::getClient()->selectCollection(self::getDatabase(), 'perftest');
```

Comment:
Is there any need to parameterize the collection name? I assume not now, and if this comes up down the line we can always switch over to a param with a default value.

Reply:
Seen you comment after merging. We will figure out if we need more that one collection.
Same for client options were not used and that I removed in the PR.